packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -508,7 +512,7 @@ export class SmoothControls extends EventDispatcher {
   }
 
   private pixelLengthToSphericalAngle(pixelLength: number): number {
-    return 2 * Math.PI * pixelLength / this.element.clientHeight;
+    return 2 * Math.PI * pixelLength / this.scene.height;
```

Comment:
I have no idea why `clientHeight` was used here; `scene.height` is correct as this is the one that is assured to never be < 1.